packages/node-cli/src/utils/AuthenticationUtils.js

```diff
@@ -6,8 +6,12 @@
 
 const FileUtils = require('./FileUtils');
 const NodeTranslationService = require('../services/NodeTranslationService');
-const { ERRORS, UTILS } = require('../services/TranslationKeys');
-const { FILES } = require('../ApplicationConstants');
+const {
+	ERRORS,
+	UTILS,
+	COMMAND_SETUPACCOUNTCI: { ERRORS: { NOT_EXISTING_AUTH_ID } },
+} = require('../services/TranslationKeys');
+const { FILES, COMMAND_AUTHENTICATE_CI_REUSE } = require('../ApplicationConstants');
```

Comment:
`COMMAND_AUTHENTICATE_CI_REUSE` it's unused. Please remove import and check we are not leaving behind some unused constant.